cli/popper/cli.py

```diff
@@ -96,4 +98,9 @@ def signal_handler(sig, frame):
         pu.info("Stopping container '{}'\n".format(container.name))
         container.remove(force=True)
 
+    for img in set(singularity_list):
+        if os.path.isfile(img):
+            pu.info('Removing {}\n'.format(img))
+            os.remove(img)
```

Comment:
here we only need to stop the container. Removal is done in the `singularity_rm` function, which is invoked based on whether the `--reuse` flag is given